pages/about.tsx

```diff
@@ -1,19 +1,66 @@
-export default function About() {
+import type { GetStaticProps, InferGetStaticPropsType } from 'next';
+interface Contributor {
+  login?: string;
+  id?: number;
+  avatar_url?: string;
+  html_url?: string;
+  type?: string;
+  site_admin?: boolean;
+  contributions?: number;
+}
+
+export const getStaticProps: GetStaticProps = async () => {
+  const res = await fetch('https://api.github.com/repos/teknologi-umum/blog/contributors');
+  const contributors: Contributor[] = await res.json();
+  return {
+    props: {
+      contributors,
+    },
+  };
+};
+
+export default function About({ contributors = [] }: InferGetStaticPropsType<typeof getStaticProps>) {
   return (
     <div>
-      <div className="mx-auto px-4 max-w-screen-lg">
-        <h1 className="text-center text-4xl font-bold my-8">Tentang Teknologi Umum</h1>
-        <p className="text-lg text-gray-700 leading-loose mb-8">
-          Consectetur veniam repellat placeat iure eveniet nobis Soluta neque ipsam aliquam veritatis reprehenderit!
-          Fuga porro ipsam itaque illo libero quas ut fugit Delectus ducimus odio optio aliquid hic. Placeat odit
-          eveniet veritatis tempore recusandae In perferendis corrupti inventore eos eum.
+      <div className="w-4/5 text-lg font-serif">
```

Comment:
lupa belom ditengahin/dilebarin atau emang dibikin segini?

![image](https://user-images.githubusercontent.com/51877647/130346130-e663dabd-7073-48e3-a5cf-7ac0a29f2c47.png)

Reply:
kok di kamu kek gitu.. tapi iya emang sengaja mau di rata kiri in. kalo di ketengahin UI wise aneh banget sama heading atas "Teknologi Umum" nya